.travis.yml

```diff
@@ -18,5 +18,5 @@ services:
   - postgresql
 
 before_script:
-  - psql -c "CREATE DATABASE bookstest;" -U postgres
-  - npm run migrations-test
+  - npm run lint
+  - npm run test
```

Comment:
this should still run the migrations, you can choose to change it latter but you will need to run them either way as some point